login-workflow/src/new-architecure/components/workflow/WorkflowCardActions.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import CardActions from '@mui/material/CardActions';
+import { Theme, useTheme } from '@mui/material/styles';
+import MobileStepper from '@mui/material/MobileStepper';
+import Button from '@mui/material/Button';
+import { useLanguageLocale } from '../../../auth-shared';
+
+type DataObject = { [key: string]: any };
+
+export type WorkflowCardActionsProps = {
+    canGoNext?: boolean | ((data: DataObject) => boolean);
+    canGoPrevious?: boolean | ((data: DataObject) => boolean);
+    showPrevious?: boolean;
+    showNext?: boolean;
+    previousLabel?: string;
+    nextLabel?: string;
+    onPrevious?: (data?: any) => void;
```

Comment:
I actually don't think we need to know about the `data` in this component. The actual screens will create a callback that passes the data where it needs to go. Do you agree?